app-web/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:8
+
+RUN apt-get update && \
+  yes | apt-get upgrade
+
+RUN mkdir /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package*.json /usr/src/app/
```

Comment:
I think this line isn't needed any more because `COPY . .` will take everything with it.

Reply:
removed it